package.json

```diff
@@ -34,11 +32,11 @@
 		"select2": "github:woocommerce/selectWoo#1.0.8",
 		"sidr": "^2.2.1",
 		"tinymce": "^4.9.11",
-		"wp-admin-colors": "^6.0.0"
+		"wp-admin-colors": "https://github.com/pressbooks/wp-admin-colors#chore/upgrade-sass"
```

Comment:
PR already merged here https://github.com/pressbooks/wp-admin-colors/pull/21. We need to release it and then bump the version before merging this one.